### PR TITLE
Log output

### PR DESCRIPTION
**Pull request by shashank-boyapally**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adds log output as a file

## Related Tickets & Documents

- Related Issue #641 
- Closes #

### CODE REVIEW

File: pkg/util/utils.go [hunk 1]
```diff
@@ -32,3 +36,13 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc, duration time.Duration,
 	}
 	return wait.ExponentialBackoff(backoff, fn)
 }
+
+func SetupLogging(uuid string) {
+	logFileName := fmt.Sprintf("logfile_%s.log", uuid)
```
Comment (rsevilla87) [inline pkg/util/utils.go:41]: Let's call prefix it with kube-burner better? sounds more intuitive

```suggestion
	logFileName := fmt.Sprintf("kube-burner-%s.log", uuid)
```

File: pkg/util/utils.go [hunk 2]
```diff
@@ -32,3 +36,13 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc, duration time.Duration,
...
+	file, err := os.OpenFile(logFileName, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```
Comment (rsevilla87) [inline pkg/util/utils.go:42]: I think is better to use os.Create (it doesn't require to specify flags or filemode), although it opens the file without the O_APPEND flag, do we need this flag @afcollins ?, uuids are supposed to be unique.

File: pkg/util/utils.go (hunk 1 above)
Comment (shashank-boyapally) [reply pkg/util/utils.go:41]: Yep suits more to our tool.

File: pkg/util/utils.go (hunk 2 above)
Comment (shashank-boyapally) [reply pkg/util/utils.go:42]: Moving this to os.Create, primarily when uuid was not included I wanted to append it to the file, did not change it. os.Create makes more sense now. let me know if we want to revert back.

**Comment by rsevilla87** [comment]
Can you add some kind of check in tests to verify the log file is being successfully created and populated?

File: pkg/util/utils.go [hunk 3]
```diff
@@ -32,3 +36,13 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc, duration time.Duration,
 	}
 	return wait.ExponentialBackoff(backoff, fn)
 }
+
+func SetupLogging(uuid string) {
+	logFileName := fmt.Sprintf("kube-burner-%s.log", uuid)
+	file, err := os.Create(logFileName)
+	if err != nil {
+		log.Fatalf("Failed to open log file: %v", err)
```
Comment (vishnuchalla) [inline pkg/util/utils.go:44]: s/open/create/

**Review by rsevilla87 — APPROVED** [review]
lgtm!

**Pull request merged**